packages/central-server/src/apiV2/import/importEntities/updateCountryEntities.js

```diff
@@ -175,9 +175,12 @@ export async function updateCountryEntities(
         country_code: country.code,
         image_url: imageUrl,
         metadata: entityMetadata,
-        attributes,
       },
     );
+
+    if (attributes !== undefined) {
+      await transactingModels.entity.updateEntityAttributes(code, attributes ? { attributes } : {});
```

Comment:
I would probably make `updateEntityAttributes` to accept `attributes` without being inside another set of `{}` so that you don't need a ternary here.
Then `updateEntityAttributes` can handle the null values